lutris/game.py

```diff
@@ -1,67 +1,40 @@
 """Module that actually runs the games."""
 
-# pylint: disable=too-many-public-methods disable=too-many-lines
+from __future__ import annotations
+
```

Comment:
Oh, can we do this?

If so, can we use it to say ``str | None`` and the like? In Python 3.7, this works in mypy but not at runtime, I read.